CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ NOTE: As semantic versioning states all 0.y.z releases can contain breaking chan
 We use *breaking :warning:* to mark changes that are not backward compatible (relates only to v0.y.z releases.)
 
 ## Unreleased
-
+- [#3423](https://github.com/thanos-io/thanos/pull/3423) Thanos Configurations file page: Display all available configuration for each component on the UI.
```

Comment:
You need to move this after the `Added` heading.

Reply:
`ui` prefix would be better fit.